InfluxDB 3 Core and Enterprise write guides (#5811)
influxdata/docs-v2: content/shared/influxdb3-write-guides/use-telegraf/csv.md

  csv_metadata_rows = 0
  csv_metadata_separators = [":", "="]
  csv_metadata_trim_set = ""
  csv_skip_columns = 0
  csv_delimiter = ","
  csv_comment = ""
  csv_trim_space = false
  csv_tag_columns = []
  csv_measurement_column = ""
  csv_timestamp_column = ""
  csv_timestamp_format = ""
  csv_timezone = ""
  csv_skip_values = []
  csv_skip_errors = false
  csv_reset_mode = "none"

[[outputs.influxdb_v2]]
  urls = ["http://{{< influxdb/host >}}"]
  token = "AUTH_TOKEN"
  organization = ""
  bucket = "DATABASE_NAME"
  content_encoding = "gzip"
```
{{% /code-placeholders %}}

Replace the following:

- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}:
  the name of the database to write data to
- {{% code-placeholder-key %}}`AUTH_TOKEN`{{% /code-placeholder-key %}}:
  your {{< product-name >}} authorization token.
  _Store this in a secret store or environment variable to avoid exposing the raw token string._

  > [!Note]
  > While in alpha, {{< product-name >}} does not require an authorization token.
  > For the `token` option, provide an empty or arbitrary token string.

  > [!Tip]
  >
  > ##### Store your authorization token as an environment variable
  >

>   > To prevent a plain text token in your Telegraf configuration file, we
>   > recommend that you store the token as an environment variable and then

Reviewer: ```suggestion
  > Avoid storing a plain text token in your Telegraf configuration file.
  > Store the token as an environment variable and then
```
